plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -952,7 +952,43 @@
 
 #variable_product_options #message,
 #variable_product_options .notice {
+	display: flex;
 	margin: 10px;
+	background-color: #FCFAE8;
+	> p {
+		width: 85%;
+	}
+	.woocommerce-add-variation-price-container {
+		width: 15%;
+		display: flex;
+		justify-content: center;
```

Comment:
We may want to use `flex-end` here? the spacing on the right looks a little weird, this seems to happen on larger screens it appears.
<img width="1105" alt="Screen Shot 2022-10-28 at 10 23 26 AM" src="https://user-images.githubusercontent.com/2240960/198607624-a07d9de5-9d34-4552-897f-a1c64f0a6a16.png">

Reply:
Good idea! I fixed it in the commit [82e59b8](https://github.com/woocommerce/woocommerce/pull/34948/commits/82e59b8bed075c9bb112288f032ce7209ef4be6d)